### ISSUE TITLE
scli server create option for bonded NIC

### ISSUE THREAD

**Issue by mbubb** [softlayer/softlayer-python#617]
Hello 

looking at the server create options - it looks as if the option for a bonded NIC is missing:

```
:....................................................:.......:
:                     port_speed                     : value :
:....................................................:.......:
:      10 Mbps Public & Private Network Uplinks      :   10  :
:     100 Mbps Public & Private Network Uplinks      :  100  :
:      1 Gbps Public & Private Network Uplinks       :  1000 :
: 1 Gbps Public & Private Network Uplinks (Unbonded) :  1000 :
:....................................................:.......:

```

1000 is in twice - is this a mistake?

yrs

Michael

**Comment by sudorandom** [comment]
Thanks for the bug report. Yes, the unbonded option is not intended to be shown. The default behaviour is to configure the links to be bonded. At the moment there's not a way to order unbonded via slcli or the hardware manager.

**Comment by sudorandom** [comment]
Closing because this was addressed in #618.
